src/LendingPool.sol

```diff
@@ -905,61 +891,49 @@ contract LendingPool is LendingPoolGuardian, Creditor, DebtToken, InterestRateMo
         address account,
         address originalOwner,
         uint256 startDebt,
-        address initiator,
         address terminator,
         uint256 surplus
     ) internal {
-        // Increase the realised liquidity for the initiator.
-        (uint256 liquidationInitiatorReward, uint256 auctionTerminationReward, uint256 liquidationFee) =
-            _calculateRewards(startDebt);
-        realisedLiquidityOf[initiator] += liquidationInitiatorReward;
+        (, uint256 auctionTerminationReward, uint256 liquidationFee) = _calculateRewards(startDebt);
 
-        // get the open openDebt, openDebt = startDebt + startDebtInterest + liquidationInitiatorReward + auctionTerminationReward + liquidationFee - bids
-        uint256 openDebt = maxWithdraw(account);
+        if (surplus > 0) {
+            // If there is surplus, all openDebt is repaid.
+            uint256 rewardsAndSurplus = auctionTerminationReward + liquidationFee + surplus;
+            // Synchronize the liquidation fee with liquidity providers.
+            _syncLiquidationFeeToLiquidityProviders(liquidationFee);
+            // Increase the realised liquidity for the terminator.
+            realisedLiquidityOf[terminator] += auctionTerminationReward;
+            // Increase the realised liquidity for the original owner.
+            realisedLiquidityOf[originalOwner] += surplus;
 
-        // Calculate the bad debt.
-        if (surplus == 0) {
-            uint256 badDebt;
-            uint256 remainder;
+            totalRealisedLiquidity = SafeCastLib.safeCastTo128(uint256(totalRealisedLiquidity) + rewardsAndSurplus);
+        } else {
+            // openDebt equals startDebt + startDebtInterest + liquidationInitiatorReward + auctionTerminationReward + liquidationFee + interests - bids.
+            uint256 openDebt = maxWithdraw(account);
             if (openDebt > auctionTerminationReward + liquidationFee) {
```

Comment:
Does this need to include the initiatorReward? Or is that already settled?

Reply:
No this is when: openDebt > auctionTerminationReward + liquidationFee -> we will pay no auctionTerminationReward and liquidationFee